### ISSUE TITLE
Wiki Missing Block Display Info

### ISSUE THREAD

**Issue by AgentCPU0** [michelegargiulo/ZenCloche#1]
I've been needing a mod like this, but I can't figure out the Block display part of the script. Some example documentation on the Wiki page would help

**Comment by michelegargiulo** [comment]
I've updated the wiki to include more information and images. Let me know if there is something else I need to explain or is not clear. It also contains more info, for the new version (waiting for approval on CF and Modrinth).


If you have a more specific question, feel free to open this issue again and ask here directly, I'll try to help

**Comment by AgentCPU0** [comment]
I was attempting to use it with Thaumic Additions to grow Vis seeds. I was successful apart from the block display. So, your mod works. it's the visuals that tripped me up. I did notice you mentioned that mod specifically in the updated Wiki, so idk if I can even fix it with a script

**Comment by michelegargiulo** [comment]
Hi, sorry for the late response. With the new version of ZenCloche I was able to specify a blockstate to the registerCrop function. Unfortunately, this is the best I could got:

![image](https://github.com/michelegargiulo/ZenCloche/assets/8409482/75a673fe-4e27-41a4-beda-4f3458bda38e)


It seems a quirk about how Thaumadditions implemented its crops, with a single, un-colorized crop, and dynamically applying the color of their respective aspect. Despite each crop having a distinct blockstate, the colorization does not seem to work inside of the Cloche. While this is still better than a purple-black cube, it is not perfect, but I find it an acceptable result, especially considering that I don't really know how to fix it (and probably out of scope for this mod).

This is the code snippet I used:
```
import mods.smokeythebandicoot.zencloche.GardenCloche;

GardenCloche.registerCrop(<thaumadditions:vis_seeds/praecantatio>, [<thaumadditions:vis_pod>.withTag({Aspect: "praecantatio"})], <thaumicaugmentation:stone:2>, <blockstate:thaumadditions:vis_crop/praecantatio>);
```
